client/ayon_flame/otio/flame_export.py

```diff
@@ -231,8 +231,15 @@ def create_otio_reference(clip_data, media_info, fps=None):
 
     otio_ex_ref_item = None
 
-    is_sequence = frame_number = utils.get_frame_from_filename(file_name)
+    # match range pattern e.g. "foo_[1001-1010].ext"
+    regex_sequence = r".*\[(?P<start_frame>\d*)-(?P<end_frame>\d*)\].?"
+    is_sequence = re.match(
```

Comment:
Just a note, I would not name the variable `is_sequence` (which indicates it does hold `bool`), but e,g, `sequence_match` instead.